templates/pulp-apache.conf.erb

```diff
@@ -1,7 +1,5 @@
 ### File managed with puppet ###
 
-Include <%= scope['apache::confd_dir'] %>/pulp-vhosts80/*.conf
-
 <Location /pulp>
```

Comment:
If Pulp is no longer serving content but it's all done by Pulpcore, is this part still useful or should the entire file be dropped?

Reply:
I left this part, as Pulp is still handling katello-agent which does require API requests to Pulp presently. This should get cleaned up along with fully dropping Pulp once the switchover has been made.